src/lib.rs

```diff
@@ -206,24 +206,30 @@ macro_rules! etw_event {
             }
         );
 
+        #[used]
         static ETW_META: $crate::_details::EventMetadata = $crate::_details::EventMetadata{
             kw: $kw,
             // TODO: Hash the callsite identity at compile time, or get Identifier to implement Ord
             identity: tracing_core::identify_callsite!(&CALLSITE),
             event_tag: $tags as u32
         };
 
+        // These two statics need to be mut to avoid compiler errors about *const EventMetadata not being Sync.
+        // It's unclear why mut avoids these errors.
```

Comment:
Why?

Reply:
I think it is because they want static const to be usable from safe code, so it must follow safe thread rules. static mut is never usable from safe code so thread rules don't have to be checked.